src/components/biggive-main-menu/biggive-main-menu.scss

```diff
@@ -0,0 +1,417 @@
+:host {
+	display: block;
+}
+
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+	font-family: 'Euclid Triangle', sans-serif;
```

Comment:
Could we use spaces instead of tabs? The tabs get very wide in the github display. Not sure if we have a set convention, but e.g. https://github.com/thebiggive/components/blob/c31188812d3a9b4335fa7a48067fcc34ef12de18/src/components/biggive-header/biggive-header.scss#L232 is using spaces.

Reply:
Thanks!